Source/control.cpp

```diff
@@ -520,12 +520,22 @@ std::string TextCmdLevelSeed(const string_view parameter)
 	    "Storybook: ", glSeedTbl[16]);
 }
 
+std::string TextCmdStreamerMode(const string_view parameter)

```

Comment:
We don't use a string_view polyfill anymore since we now require C++17 (C++20).
I'm also pretty sure that string_view basically is the const version of a string, so that probably don't make sens with the const on top of it.